api_tests/lib_sdk/actors/actor.ts

```diff
@@ -177,17 +188,38 @@ export class Actor {
 
         if (isLightning) {
             this.logger.debug(`Initialising lightning for ${this.config.name}`);
-            await this.wallets
-                .getWalletForLedger("lightning")
-                .addPeer(to.wallets.getWalletForLedger("lightning"));
+            const thisLightningWallet = this.wallets.getWalletForLedger(
+                "lightning"
+            );
+            const toLightningWallet = to.wallets.getWalletForLedger(
+                "lightning"
+            );
+
+            await thisLightningWallet.addPeer(toLightningWallet);
+
+            if (this.alphaLedger.name === "lightning") {
+                // Alpha Ledger is lightning so Alice will be sending assets over lightning
+                const quantity = parseInt(this.alphaAsset.quantity, 10);
+                await thisLightningWallet.openChannel(
+                    toLightningWallet,
+                    quantity
+                );
```

Comment:
📣
My LN knowledge is a bit rusty so please excuse if this is wrong: 
I think that you can't fully _empty_ a LN channel but there has always to remain some satoshis left. This means, you will need to open the channel with a higher balance than what you want to send later on.

(There is an attack vector if this would be possible).

Reply:
Ignore this if your tests run.

Reply:
Yes good point, it is not an issue right now as we choose an arbitrary number for the invoice (20000) but it would have been one once we actually implement the swap.Well done.

Reply:
> Ignore this if your tests run.

Yes see my comment above, it works now but it would have been an issue once someone try to actually use this for a swap.